rllib/evaluation/env_runner_v2.py

```diff
@@ -1,8 +1,8 @@
+from collections import defaultdict
 import logging
 import time
-from collections import defaultdict
+import tree
```

Comment:
Add: `  # pip install dm_tree` for clarifying that this is NOT the `pip install tree` module (this also exists :/ ).

Reply:
done.